base.js

```diff
@@ -301,6 +301,7 @@ module.exports = {
     'no-lone-blocks': 'error', // https://eslint.org/docs/rules/no-lone-blocks
     'no-lonely-if': 'warn', // https://eslint.org/docs/rules/no-lonely-if
     'no-loop-func': 'error', // https://eslint.org/docs/rules/no-loop-func
+    'no-loss-of-precision': 'warn', // https://eslint.org/docs/rules/no-loss-of-precision
```

Comment:
is there a reason this rule was added with the support of V8. Shouldn't rule changes be part of a different version and then added as a dedicated feature and documented in the changelog

Reply:
Removed the rule @bob606